backend/app/api/v1/endpoints/ocp/graph.py

```diff
@@ -328,7 +328,7 @@ async def getBurnerCPUResults(uuids: list, namespace: str, index: str ):
 
 async def getBurnerResults(uuid: str, uuids: list, index: str ):
     if len(uuids) > 1 :
-        if len(uuid) > 0 :
+        if len(uuid) > 0 and uuid in uuids:
```

Comment:
It would have been nice to have a comment explaining exactly what the original logic was intended to do. `len(uuid) > 0` seems odd ... that suggests the caller is allowed to pass an empty string or a uuid value, but not `None`. Why?

Anyway, clearly someone's passing a uuid that's not in the list (which also seems logically a bit odd), and your protection seems valid and necessary for that case.

Reply:
@jtaleric any recollection?

Reply:
Well, here it is, in the `trend` API:

```python
oData = await getBurnerResults("",current_ids,index)
```

Odd to have written it that way. Two of the calls use a direct FastAPI URL parameter typed as `str`, so Pydantic won't allow an empty value, while the third call is this literal `""` which could as easily be `None`. But maybe there was a reason at the time.

Reply:
Ultimately ... I think the interface here is awkward, and could be improved, but doesn't need to be decided or implemented in this PR. I'm going to resolve the conversation. If we don't get any contrary feedback by the end of tomorrow (Thursday), I'll merge the fix so we can mark the Jira done this sprint. 🥳